core/types.go

```diff
@@ -49,3 +51,10 @@ type Processor interface {
 	// the processor (coinbase) and any included uncles.
 	Process(block *types.Block, statedb *state.StateDB, cfg vm.Config) (types.Receipts, []*types.Log, uint64, error)
 }
+
+// bor acts as a way to be able to type cast consensus.Engine;
+// since importing "github.com/maticnetwork/bor/consensus/bor" results in a cyclic dependency
+type bor interface {
```

Comment:
name could be `consense` instead of bor? interface being the verbe here.

Reply:
Removed this altogether. Not the right place to be anyway.